### DISCUSSION TITLE
Email Attachments location

### DISCUSSION THREAD

**Discussion by Bappamaiti** [protz/thunderbird-conversations#1695] [Q&A] [answered]
Hello Devs,
Good Day!

I am using this Add-on from past few months and found it as a must have tool for Thunderbird and Thank you so much for this.

Since last month, I am getting some of the emails having many conversations as reply or as forward mail. And to access the attachments I need to scroll to the end of the conversations each time. Due to this I get frustrated sometimes, and so I was looking for a way to bring them at the top or start of the conversation . 

Please guide me if it already have an option to do that or please take it as a feature request to add in future versions.

Thanks!

**Comment by Standard8** [comment, marked as answer]
This is #1366 which I hope to get to soon. We used to have something but it got lost in some of the code re-architecturing. I'm hoping to be able to bring back what we had before which was a note at the top of the message that you could click on and go straight to the attachments.
